sorc/ncep_post.fd/FDLVL.f

```diff
@@ -308,6 +319,7 @@ SUBROUTINE FDLVL(ITYPE,TFD,QFD,UFD,VFD,PFD,ICINGFD)
 !             DO 222 IFD = 1, NFD
               DONEH=.FALSE.
               DONEV=.FALSE.
+              IF(ZMID(I,J,LM) == SPVAL)CYCLE
```

Comment:
@WenMeng-NOAA   Since the range of L = LLMH, 1, -1 is considered in the code block that follows, could there be a need to reference ZMID at (I,J,**LLMH**), in place of (I,J,**LM**), for the CYCLE statement?

Reply:
@ChristopherHill-NOAA LLMH is actually assigned from values of LM. I will adopt your suggestion for code consistency.